app/src/main/java/org/simple/clinic/user/LoggedInUserPayload.kt

```diff
@@ -30,5 +30,8 @@ data class LoggedInUserPayload(
     val createdAt: Instant,
 
     @Json(name = "updated_at")
-    val updatedAt: Instant
+    val updatedAt: Instant,
+
+    @Json(name = "teleconsult_phone_number")
+    val teleconsultPhoneNumber: String?
```

Comment:
IIRC, we are also going to make other changes to the user model and add the "capabilities" property. Can we try to make this part of the same migration?

Reply:
Yes it has `capabilities` as well. But I wasn't sure it was confirmed and it was listed as a separate story. Should I go ahead with that or wait until all the API contracts are confirmed? I picked this up because it's the most simple API contract.

Reply:
Hmm, alright. Let's add another migration for that once it's finalized.